keyboards/wonderland/wonderland.c

```diff
@@ -0,0 +1,40 @@
+#include "wonderland.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
+
+void led_init_ports(void) {
+    // * Set our LED pins as output
+    DDRB |= (1 << 1) | (1 << 2) | (1 << 3);
+}
+
+void led_set_kb(uint8_t usb_led) {
+	if (usb_led & (1 << USB_LED_NUM_LOCK)) {
+        DDRB |= (1 << 1);
```

Comment:
```suggestion
        setPinOutput(B1);
```

Reply:
Actually, should these not be `writePinHigh`/`writePinLow`?

Reply:
Is there a doc for new convention?

Reply:
Looks like it should writePinLow and writePinHigh, will fix. https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_set_user-implementation

Reply:
Not sure if I'd call it documentation, but... https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_set_user-implementation

Reply:
The macros to use depend on how the LED is wired; that example assumes the pins are initialised to output (which is the case with this board).

Though, I am not sure if initing to input is electrically valid for such a circuit. Maybe switching between tristate and input low can toggle the LED in current sink configuration...